src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/StringCredentials.java

```diff
@@ -9,7 +9,7 @@
 @Describable("Secret text")
 public class StringCredentials extends BaseStandardCredentials {
 
-    public Control secret = control("secret");
+    public Control secret = control(by.name("_.secret"));
```

Comment:
Don't you want `control(by.name("_.secret"), by.name("secret"));` to keep compatibility with previous versions.